Lesson27-Loops&Conditions/scripts/main.js

```diff
@@ -0,0 +1,301 @@
+        question = prompt('Do you want to see the next day? Enter Yes or No')
+    }
+    if (question == 'yes') {
+        nextDay.splice(0, 1, 'Friday')
+        console.log (nextDay)
+        question = prompt('Do you want to see the next day? Enter Yes or No')
+    }
+    if (question == 'yes') {
+        nextDay.splice(0, 1, 'Saturday')
+        console.log (nextDay)
+        question = prompt('Do you want to see the next day? Enter Yes or No')
+    }
+    if (question == 'yes') {
+        nextDay.splice(0, 1, 'Sunday')
+        console.log (nextDay)
+        question = prompt('Do you want to see the next day? Enter Yes or No')
+    }
+    else (question == 'no'); {
+        console.log('We`re done!')
+    }
+}
+// loopDay()
+
+// 9. Display the multiplication table of all numbers from 2 to 9. Every number has to be multiplied by 1 to 10. 
+function multiplication() {
+    let x = 2
+    for (x=2; x <= 10; x++) {
+        for (i=0; i<=10; i++) {
+            let sum = i * x
+            console.log(sum)
+        }
+    }
+}
+// multiplication()
+
+
+// 10. “Guess the number” game. 
+// Suggest a user to think of a number from 0 to 100 and guess it in the following way: 
+// every loop iteration needs to divide the range of numbers in half, 
+// you represent the result as N and ask the user “Your number > N, < N or == N?” 
+// Depending on what the user said, narrow down the range. So the starting range is 0 to 100, divided in half is 50. 
+// If the user said the number is > 50, the next range is 51 to 100, and so on, until the user chooses == N. 
+
+function guessGame() {
+    let userNumber = alert('Think of a number between 0 and 100!'); 
+    let guess = prompt('Is your number > 50 < 50 or == 50 ? Enter: <, > or ==:');
+    let min = 0;                             
+    let middle = 50;                          
+    let max = 100;                              
+    while (guess != '==') {    
+        if (guess == '>') {                        
+            min = middle;                      
+            range = max - min;                     
+            rangeHalf = range / 2;                  
+            rangeHalf = Math.floor(range / 2);      
+            let center = min + rangeHalf;           
+            middle = center;   
+            let guessAgain = prompt('Is your number > ' + middle + ' < ' + middle + ' or == ' + middle + ' ? Enter: <, > or ==:'); 
+            guess = guessAgain;
+        }
+        if (guess == '<') {                                
+            max = middle                               
+            range = max - min;                             
+            rangeHalf = range / 2;                         
+            rangeHalf = Math.floor(range / 2);             
+            let center = min + rangeHalf;                   
+            middle = center;
+            let guessAgain = prompt('Is your number > ' + middle + ' < ' + middle + ' or == ' + middle + ' ? Enter: <, > or ==:'); 
```

Comment:
You can use next syntax
```
 let guessAgain = prompt(`Is your number > ${middle} < ${middle} or == ${middle}  ? Enter: <, > or ==:`); 
```

Reply:
I've done this on more resent code and will update it on my pervious code. 
Thank you!